onnxruntime/contrib_ops/cpu/attnlstm/attention_wrapper.cc

```diff
@@ -34,14 +35,14 @@ AttentionWrapper<T>::AttentionWrapper(AllocatorPtr alloc, const logging::Logger&
 
 // rnn_cell_output is of [batch_size, rnn_cell_hidden_size]
 template <typename T>
-void AttentionWrapper<T>::ProcessOutput(const gsl::span<const T>& rnn_cell_output) {
+void AttentionWrapper<T>::ProcessOutput(const gsl::span<const T>& rnn_cell_output, concurrency::ThreadPool* tp) {
```

Comment:
>concurrency::ThreadPool* tp [](start = 83, length = 27)

This should take a reference as we need a valid value for the code to work

Reply:
Yes. It's nullable.